### ISSUE TITLE
Show widget on single course page

### ISSUE THREAD

**Issue by donnapep** [woocommerce/sensei-course-progress#93]
**Is your feature request related to a problem? Please describe.**
Currently, the course progress widget is only displayed on the module and lesson pages. This means that when a learner navigates to the single course page to resume a course, they have to scroll to the bottom of the page in order to see where they've left off.

**Describe the solution you'd like**
Display the course progress widget on the single course page.

**Comment by clemsos** [comment]
Yes, will be very nice to have. 

This could also be an option on the widget admin panel with a checkbox "show on single course page"

**Comment by clemsos** [comment]
@donnapep will you accept a PR for this?

**Comment by donnapep** [comment]
@clemsos Yup, would be happy to review a PR. I would opt for showing it by default on that page though. We could add a setting later if people ask for one.
